### DISCUSSION TITLE
Decimal sign changes depending on decimal value

### DISCUSSION THREAD

**Discussion by McOrion** [firefly-iii/firefly-iii#7575] [Firefly III]
When viewing the budgeted amount in Firefly-iii I see some strange behavior with the decimal sign. When I have a budget in whole euro's, I have a dot as decimal sign (e.g.: € 100.00), but as soon as the budget has a decimal value, the decimal sign changes to a comma (e.g.: € 100,01). This is only the situation on the budget page with the budgeted amount. On all other places (budget spend, contracts, transaction) all decimal signs are a comma, regardless if the amount is in whole euro's or with a decimal value.

I'm running Firefly-iii v6.0.11 on a self-hosted server (Debian) with a MariaDB database on a separate Debian server. Both servers have en_US as locale. Firefly has Dutch as language and locale. I use the Firefox browser with Dutch as language.

Is this still related to some language setting? It's annoying at most, but makes me wonder. 

![Budget_1](https://github.com/firefly-iii/firefly-iii/assets/59847229/872bfe59-4f5b-4d02-84f5-779cc39a1eac)
![Budget_2](https://github.com/firefly-iii/firefly-iii/assets/59847229/4aea8b55-0fef-43b6-8cd0-c45caebe0709)

**Comment by JC5** [comment]
I've spotted it as well. In both cases, Firefly III sends the same format to the browser: xxx.xx. I have no clue if it's something I can help.

**Comment by McOrion** [comment]
I've tried it with Edge and Chrome, and there the decimal sign is shown as a comma, in all cases. So to me, it looks like a Firefox issue then. Thanks for spotting it too, now I know I'm not the only one ;-)
